packages/dart_frog_test/lib/src/dart_frog_test_context.dart

```diff
@@ -0,0 +1,53 @@
+import 'package:dart_frog/dart_frog.dart';
+import 'package:mocktail/mocktail.dart';
+
+class _MockRequestContext extends Mock implements RequestContext {}
+
+/// {@template dart_frog_test_context}
+/// A context used to test Dart Frog handlers.
+/// {@endtemplate}
+class DartFrogTestContext {
```

Comment:
**Nit:** I was thinking about renaming this to `TestRequestContext` the other day, but I was unsure. I prefer reading `TestRequestContext` over `DartFrogTestContext`. I think so because in the README is says:

> This class makes it simple to mock a `RequestContext`

Do you foresee it having a different usage than to simply "mock a `RequestContext`"?

```suggestion
class TestRequestContext {
```

Reply:
Hum, I don't think so! I think this is a good suggestion, I am going to rename it